lib/diagnostic.js

```diff
@@ -6,28 +6,62 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (word) {
+ return word.replace(/\w+/,'').toUpperCase();
 };
 
+let normalizedWords = words.map(normalize);
+};
+
+comfort : 2
+clarity :3
+
+I used class notes from yesterday afternoon.
+
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+const uniqueWords = function () {
+  let uniqueWordsAsKeys = function (array) {
+    let result ={};
+    array.forEach(function (word){
+      result[word] = null;
+    })
+
+    return result;
+  };
 };
 
```

Comment:
What is the point of having a function within a function here? Try to avoid declaring them on top of each other.